src/index.html

```diff
@@ -10,7 +10,32 @@
     <title>Search bar airbnb</title>
     <link rel="stylesheet" href="./style.css">
   </head>
-  <body>
-    <h1>Search bar airbnb</h1>
+  <body class="body">
+    <form
+      action="search-big"
+      method="post"
+      data-qa="big"
+    >
+      <input
+        class="search-big"
+        type="text"
+        name="searchOne"
+        placeholder="Try &quot;Los Angeles&quot;"
+        data-qa="keypress"
+      >
+    </form>
+
+    <form
+      action="search-small"
+      method="post"
+      data-qa="small"
+    >
+      <input
+        class="search-small"
+        type="text"
+        name="searchTwo"
+        placeholder="Try &quot;Los Angeles&quot;"
+      >
+    </form>
```

Comment:
```suggestion
  <body class="body">
    <form
      action="search-big"
      method="post"
      data-qa="big"
    >
      <input
        class="search"
        type="text"
        name="searchOne"
        placeholder="Try &quot;Los Angeles&quot;"
        data-qa="keypress"
      >
    </form>

    <form
      action="search-small"
      method="post"
      data-qa="small"
    >
      <input
        class="search search-small"
        type="text"
        name="searchTwo"
        placeholder="Try &quot;Los Angeles&quot;"
      >
    </form>
```
it should work like this

Reply:
2nd point in the checklist

Reply:
Fix it